nemo_skills/inference/model/openai.py

```
@@ -108,6 +108,8 @@ def _build_chat_request_params(
             raise ValueError("`min_p` is not supported by OpenAI API, please set it to 0.0.")
         if stream and top_logprobs is not None:
             raise ValueError("`top_logprobs` is not supported with stream=True.")
+        if extra_body:
+            raise ValueError("`extra_body` is not supported by OpenAI API")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)